generators/Exercises/RnaTranscription.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Generators.Input;
+using Generators.Output;
+
+namespace Generators.Exercises
+{
+    public class RnaTranscription : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var canonicalDataCase in CanonicalData.Cases)
+            {
+                canonicalDataCase.Expected = canonicalDataCase.Expected ?? new UnescapedValue("null");
```

Comment:
This is not ideal and should not be necessary. A `null` value should be displayed as `null`. I'll try and fix that.